src/BUILD

```diff
@@ -318,16 +323,19 @@ cc_binary(
 cc_binary(
     name = "libcustom_node_add_one.so",
     srcs = [
-        "custom_nodes/add_one/add_one.cpp",
-        "custom_node_interface.h",
-        "queue.hpp",
+        "custom_nodes/common/utils.hpp",
         "custom_nodes/common/buffersqueue.hpp",
         "custom_nodes/common/buffersqueue.cpp",
         "custom_nodes/common/customNodeLibraryInternalManager.hpp",
```

Comment:
Stick to our style guideline:
```
customNodeLibraryInternalManager.hpp
vs
custom_node_library_internal_manager.hpp
```

Reply:
I have renamed customNodeLibraryInternalManager.hpp and customNodeLibraryInternalManager.cpp files.